content/management/index.mdx

```diff
@@ -8,20 +8,125 @@ desc: "There are dedicated APIs for the resource management. Use the Configurati
+
+</Warning>
+
+# Resources
+
+## Agents
+
+An agent is a type of user who communicates with customers. You can see the agent data structure [here](/messaging/agent-chat-api/#agent). The Configuration API allows for multiple agent-related actions, including creating, updating, deleting, and suspending agents.
+
+<SectionLink to={"/management/configuration-api/#agents"}>
+  See agent methods
+</SectionLink>
+
+## Auto access
+
+Auto access is a functionality that allows for assigning customers to dedicated groups based on the URL or the customer's geolocation.
+
+<SectionLink to={"/management/configuration-api/#auto-access"}>
+  See auto access methods
+</SectionLink>
+
+## Bots
+
+Bots are similar to regular agents. The main difference between them is that bots have a greater potental for automation, for example, they can listen for and react to incoming webhooks and pushes.
+
+Bots are created and managed via the Configuration API. Just as regualar agents, bots can call Agent Chat API by the Web or RTM API.
+
+Bots are authorized with the use of the [agent token](/authorization/authorizing-api-calls/#agent-authorization-flows). Using bots requires sending the `X-Author-Id` header for the Web API and the `author_id` property for the RTM API.
+
+<SectionLink to={"s/management/configuration-api/#bots"}>
+  See bot methods
+</SectionLink>
+
+## Groups
+
+Groups let you organize your work by creating teams with agent and bot members. You can use groups to do configuration based on shared settings (language, working hours, department) and to separate chat routing.
+
+<SectionLink to={"/management/configuration-api/#groups"}>
+  See group methods
+</SectionLink>
+
+## Properties
+
+Properties are key-value storages. Depending on the API version, they can be set within the following locations: a chat, a thread, an event, a group, and a license. Property configuration is owned by a Client Id (integration), not by license (v3.3+). Properties can be public or private.
+
+<SectionLink to={"/management/configuration-api/#properties"}>
+  Learn more about properties
+</SectionLink>
+
+<SectionLink to={"/management/configuration-api/#methods-4"}>
+  See property methods
+</SectionLink>
+
+## Webhooks
+
+In a nutshell, actions invoked by the use of Messaging APIs or the Configuration API result in events. You can be notified about those events with webhooks. LiveChat provides a number of webhooks, which you can register and manage via the Configuration API. Just like properties, webhooks are registered per Client Id (integration), not per license (v3.3+).
+
+We can distinguish two types of webhooks:
+
+- license webhooks
+- bot webhooks
+
+Once **license webhooks** are set up, you will always receive them. Read our tutorial on <a href="https://developers.livechat.com/updates/setting-up-webhooks-with-livechat-api/" target="_blank">how to configure LiveChat license webhooks via the Configuration API</a>.
```

Comment:
by setting up you meain registering and **enabling** them, right?

Reply:
Yeah :) I think it's okay for a general overview because a developer might want to "do something with properties, some configuration/setup), and doesn't really know that props need to be registered & enabled.

Reply:
ok